src/main/webapp/creatives/maps/simid_map_creative.test.js

```diff
@@ -0,0 +1,68 @@
+/** Tests to see if Find Nearest X button is appending correct text */
+
+import SimidMapCreative from './simid_map_creative.js';
+
+let testMap;
+
+beforeEach(() => {
+    testMap = new SimidMapCreative();
+});
+
+test('testing button text updates from ad params', ()=> {
```

Comment:
super minor nit.
`() => {`

for the test below also

Reply:
fixed